viewer/packages/360-images/src/annotation/ImageAnnotationObject.ts

```diff
@@ -32,21 +36,32 @@ export class ImageAnnotationObject implements Image360Annotation {
 
   public static createAnnotationObject(annotation: AnnotationModel, face: FaceType): ImageAnnotationObject | undefined {
     const detection = annotation.data;
-    assert(isAnnotationsObject(detection));
 
-    let objectData: ImageAnnotationObjectData;
+    const objectData = ImageAnnotationObject.createObjectData(detection);
 
-    if (detection.boundingBox !== undefined) {
-      objectData = new BoxAnnotationData(detection);
-    } else if (detection.polygon !== undefined) {
```

Comment:
Is it possible that both a box and polygon is defined? Wouldnt we then want the polygon?

Reply:
I think it's meant to never happen, but could probably happen due to user errors :thinking: Good catch, I'll change it